D89 get_dq_cond/get_dq_evap instrumentation. Generated by diff -u against the PRISTINE source (2026-10-05); anchored in unchanged code, so it applies to the pristine file or after any other ATM_DRV.f patch (verified after ATM_DRV.f.patch: offset 5).

--- CLOUDS2.F90.orig	2024-05-21 16:10:29.000000000 -0400
+++ CLOUDS2.F90	2026-10-05 13:39:59.957283016 -0400
@@ -1361,6 +1361,7 @@
             CCM(L-1)=MPLUME
 
             !**** CONDENSE VAPOR IN THE PLUME AND ADD LATENT HEAT
+            call ffdq_site(1)
             call get_dq_cond(smp,qmp,plk(l),mplume,lhx,pl(l),dqsum,fqcond)
 
             if(DQSUM.gt.0. .and. QMP.gt.teeny) then
@@ -2064,6 +2065,7 @@
             SLH=LHX*BYSHA
             TNX1=SMDN*PLK(L)/DDRAFT   ! save for tracers
 
+            call ffdq_site(2)
             call get_dq_evap(smdn,qmdn,plk(l),ddraft,lhx,pl(l),cond(l), dqsum,fqcond1)
 
             !**** EVAPORATE CONVECTIVE CONDENSATE IN DOWNDRAFT AND UPDATE DOWNDRAFT
@@ -2713,6 +2715,7 @@
 
           if (PRCP.gt.0.) then
 
+            call ffdq_site(3)
             if (mcloud.gt.0) call get_dq_evap (smold(l),qmold(l),plk(l),airm(l),lhx,pl(l),prcp*AIRM(L)/MCLOUD, dqsum,fprcp)
             dqsum=dqsum*MCLOUD*BYAM(L)
 
@@ -3999,6 +4002,7 @@
         QHEATI(L)=0.
         if (LHX.eq.LHE.and.QCLX(L).gt.0.) then
 
+            call ffdq_site(4)
           call get_dq_evap (tl(l)*rh00(l)/plk(l),ql(l)*rh00(l),plk(l),rh00(l),lhx,pl(l),qclx(l)/(fssl(l)*rh00(l)), dqsum,fqcond1)
           DWDT=DQSUM*RH00(L)*FSSL(L)
 
@@ -4010,6 +4014,7 @@
         end if
         if (LHX.eq.LHS.and.QCIX(L).gt.0.) then
 
+            call ffdq_site(5)
           call get_dq_evap (tl(l)*rh00(l)/plk(l),ql(l)*rh00(l),plk(l),rh00(l),lhx,pl(l),qcix(l)/(fssl(l)*rh00(l)), dqsum,fqcond1)
           DWDT=DQSUM*RH00(L)*FSSL(L)
 
@@ -4422,6 +4427,7 @@
       if(RH1(L).gt.1.) then    ! RH was used in old versions
         SLH=LHX*BYSHA
 
+            call ffdq_site(6)
         call get_dq_cond(tl(l),ql(l),1d0,1d0,lhx,pl(l),dqsum,fcond)
 
         if(DQSUM.gt.0.) then
@@ -7292,6 +7298,7 @@
     FCOND=DQSUM/QM
   end if
 
+  call ffdump_dq(1,sm,qm,plk,mass,lhx,pl,0d0,dqsum,fcond)
   return
 end subroutine get_dq_cond
 
@@ -7332,5 +7339,6 @@
     FEVP=DQSUM/COND
   end if
 
+  call ffdump_dq(2,sm,qm,plk,mass,lhx,pl,cond,dqsum,fevp)
   return
 end subroutine get_dq_evap
